internal/pprofserver/pprofserver.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pprofserver
+
+import (
+	"flag"
+	"net/http"
+	_ "net/http/pprof" // Needed to enable the performance profiler
+	"strconv"
+
+	"go.uber.org/zap"
+)
+
+const (
+	httpPprofPortCfg = "http-pprof-port"
+)
+
+// pport is used to ensure that if AddFlags was called the setup works even if viper
+// was not set.
+var pport *uint
```

Comment:
I think we should use a `viper` over a global var, even if we have to do some extra work in the ocagent/main.go and ocagent/cli.go

Reply:
I started with that but as you saw it requires more changes to the agent, so I simplified. Let me see the cost of merging cli.go and main.go and fix both things at once.

Reply:
Let's do it separate it starts to pull a lot of other changes, it will be better to separate in a change to move agent to viper.

Reply:
Can we implement this with viper though and then have the agent do the extra work to pass the viper? I believe it is similarly done for the FromViper that the receivers are built with

Reply:
that way we wont need to change as well when we go to viper

Reply:
ok, it won't be pretty but seems reasonable for now

Reply:
done